### DISCUSSION TITLE
Geopackage validation

### DISCUSSION THREAD

**Discussion by Mi-I** [INSPIRE-MIF/helpdesk-validator#958] [Q&A]
Hi,
please excuse my "beginner level" question ( I am not tech person but a coordinator at my work place)

We would like to start migrating from gml files to geopackage format for our Inspire files. How do we validate the inspire-geopackage files? The ETF validator we usually use does only take GML. 

Thanks in advance.

**Comment by fabiovinci** [comment]
Dear @Mi-I,

the validation of GeoPackage files is currently not expected to be integrated into the Reference Validator.

According to the [INSPIRE Good Practice: GeoPackage encoding of INSPIRE datasets](https://github.com/INSPIRE-MIF/gp-geopackage-encodings/blob/main/spec/GeoPackage_Good_Practice_initiation_fiche.md#description-of-the-gp) if you want to use the GeoPackage as an alternative encoding for INSPIRE data sets you must provide the following resources:

- A description of the logical model
- An empty GeoPackage template file
- A formal, executable specification of the model transformation referencing and conforming to the generic INSPIRE model transformation rules
- A formal, executable model for data transformation that allows to transform a data set encoded as GeoPackage to the default encoding.

So, you can test the INSPIRE compliance using the GML file generated by the data transformation model you provided.

I hope that this helps to clarify.

**Reply by Mi-I** [reply]
Thank you for the advise. 

I have also received few other advises thru other channels so we feel we have better understanding.
You mention "provide". Shall it be posted somewhere for official approval?
